### PR TITLE
Taylor editing user profile

### PR DESCRIPTION
**Pull request by AmazingOrange**

### CODE REVIEW

File: app/html/caller_profile.html [hunk 1]
```diff
@@ -194,19 +194,23 @@ <h1>User profile</h1>
         <div id="user_profile_section">
           <img width="200" src="/imgs/saveme.jpg" alt="">
           <p>First Name: </p>
-          <div id="detail_user_firstN">bank</div>
+          <div class="edit" id="detail_user_firstN" >bank</div>
           <p>Last Name: </p>
-          <div id="detail_user_lastN">name</div>
+          <div class="edit" id="detail_user_lastN" >name</div>
           <p>Eamil: </p>
```
Comment (mksample) [inline app/html/caller_profile.html:200]: Small spelling mistake

File: app/html/caller_profile.html [hunk 2]
```diff
@@ -194,19 +194,23 @@ <h1>User profile</h1>
...
-          <div id="detail_user_email">email</div>
+          <div class="edit" id="detail_user_email" >email</div>
```
Comment (mksample) [inline app/html/caller_profile.html:201]: Don't need any changes here, just thought you could use this info. Right now the backend doesn't allow non-admin users to change their email and it won't show any errors if you try to submit an updated email. So if changes are made here they will not be reflected in the DB and no error will show. This is fine for now but the more people that know about it the better :)

File: app/html/caller_profile.html [hunk 3]
```diff
@@ -255,6 +259,7 @@ <h1>Settings</h1>
   </div>
   <!-- SCRIPT FOLDER-->
   <script src="/js/caller_profile.js"></script>
+  <script src="/js/caller_profile_edit.js"></script>
```
Comment (mksample) [inline app/html/caller_profile.html:262]: Nice idea on the second script, should probably do this myself...

File: app/html/create_user.html [hunk 4]
```diff
@@ -60,8 +60,8 @@ <h1>SaveMe Sign Up</h1>
       </div>
       <span id="createUserStatus" class="status"></span>
       <hr>
-      <input type="button" class="cancelButton" id="cancelButton" value="Cancel" />
-      <input type="button" class="signUpButton" id="signUpButton" value="Sign up" />
+      <button class="cancelButton" id="cancelButton">Cancel</button>
```
Comment (mksample) [inline app/html/create_user.html:63]: Button one to be reverted

File: app/html/create_user.html [hunk 5]
```diff
@@ -60,8 +60,8 @@ <h1>SaveMe Sign Up</h1>
...
+      <button class="signUpButton" id="signUpButton">SIGN UP</button>
```
Comment (mksample) [inline app/html/create_user.html:64]: Button two to be reverted

**Review by mksample — CHANGES_REQUESTED** [review]

File: app/html/caller_profile.html (hunk 2 above)
Comment (AmazingOrange) [reply app/html/caller_profile.html:201]: I was not sure about letting users change their email address. Yeah, I can make email to be disabled for changing.

File: app/html/caller_profile.html (hunk 1 above)
Comment (AmazingOrange) [reply app/html/caller_profile.html:200]: oophs eamil, I replace it with email

**Review by mksample — APPROVED** [review]
LGTM!

**Pull request merged**